html/components/masthead.js

```diff
@@ -26,9 +26,9 @@ let direction = scrollYDirection();
 const toggleMenu = (scrollDirection) => {
   const masthead = document.querySelector('[data-sprk-masthead]');
   if (scrollDirection === 'down') {
-    masthead.classList.add('sprk-c-Masthead--hidden');
+    masthead.classList.add('sprk-c-Masthead--is-hidden');
```

Comment:
I had created these new classes as part of the angular api upgrade for masthead and planned on updating html as part of the react masthead work. Some of those were needed now, the hidden classes etc. So i just updated parts of masthead that made sense for this as well.